### PR TITLE
Sandbox/Production environment config

### PR DESCRIPTION
**Pull request by yen-tt**
- add commented out `environment` in global_config.json

J=SLAP-1663
TEST=manual

see the environment is change based on value in global_config.json

### CODE REVIEW

File: test-site/config/global_config.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   // "token": "<REPLACE ME>", // The auth token to access Answers experience.
   "apiKey": "2d8c550071a64ea23e263118a2b0680b", // The answers api key found on the experiences page. This will be provided automatically by the Yext CI system
   // "experienceVersion": "<REPLACE ME>", // the Answers Experience version to use for API requests. This will be provided automatically by the Yext CI system
+  //"environment": "<REPLACE ME>", // The environment to run on for this Answers Experience. (e.g. 'production' or 'sandbox')
```
Comment (oshi97) [inline test-site/config/global_config.json:6]: space after double after the double slash

Comment (yen-tt) [reply test-site/config/global_config.json]: yea

**Review by oshi97 — APPROVED** [review]
so simple haha

File: global_config.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   // "token": "<REPLACE ME>", // The auth token to access Answers experience.
   // "apiKey": "<REPLACE ME>", // The answers api key found on the experiences page. This will be provided automatically by the Yext CI system
   // "experienceVersion": "<REPLACE ME>", // the Answers Experience version to use for API requests. This will be provided automatically by the Yext CI system
+  // "environment": "<REPLACE ME>", // The environment to run on for this Answers Experience. (e.g. 'production' or 'sandbox')
```
Comment (oshi97) [inline global_config.json:6]: maybe `Must be 'production' or 'sandbox'`? do we support other options?

Comment (oshi97) [reply global_config.json:6]: Or even just `i.e.` instead of `e.g.`

Comment (yen-tt) [reply global_config.json:6]: hmm from the looks of it (i.e. in `urlutils.js`), I only know we have production and sandbox endpoints. So I can update the comment as you suggested.

**Review by oshi97 — APPROVED** [review]

File: global_config.json [hunk 3]
```diff
@@ -3,6 +3,7 @@
   // "token": "<REPLACE ME>", // The auth token to access Answers experience.
   // "apiKey": "<REPLACE ME>", // The answers api key found on the experiences page. This will be provided automatically by the Yext CI system
   // "experienceVersion": "<REPLACE ME>", // the Answers Experience version to use for API requests. This will be provided automatically by the Yext CI system
+  // "environment": "<REPLACE ME>", // The environment to run on for this Answers Experience. (i.e. 'production' or 'sandbox')
```
Comment (oshi97) [inline global_config.json:6]: instead of `REPLACE ME` should the default value just be "production"?

Comment (yen-tt) [reply global_config.json]: good point. updated!

**Review by oshi97 — APPROVED** [review]

**Pull request merged**